change/@fluentui-react-charting-effaa597-2236-4e8c-b8f3-91a2422c9f44.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Optimizing code for perfromance improvement",
```

Comment:
Test scenarios like duplicate data for area chart, single data point present, large number of datapoints present, many legends present, legend overflow.
Lets give this PR to our testers to test E2E scenarios like hover card show/hide, legend hover/select/unselect, accessibility.

Reply:
Done